content/docs/ops/runbook/rotating-bosh.md

```diff
@@ -0,0 +1,194 @@
+---
+menu:
+  docs:
+    parent: runbook
+
+title: Rotating Secrets III - BOSH
+---
+
+## Introduction
+
+Rotating BOSH secrets are pretty straight forward and should be done first.
+There are known limitations of the BOSH agent when updating the main BOSH
+password for each deployment. Deployments of BOSH should be done serially due to
+the dependencies of where these BOSH directors are being deployed to. Please
+reference the [18F/cg-deploy-bosh](https://github.com/18F/cg-deploy-bosh)
+repository along with the rest of this guide.
+
+### Agent limitations when updating BOSH
+
+The following should be referenced throughout the various deployments. [According
+to the BOSH development team](https://cloudfoundry.slack.com/archives/C02HPPYQ2/p1492784585438034),
```

Comment:
https://github.com/cloudfoundry/bosh/issues/1140 Found an oldie related to changing nats passwords. Will link to that.